app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -0,0 +1,59 @@
+package fr.free.nrw.commons.feedback;
+
+import android.content.Context;
+import fr.free.nrw.commons.feedback.Feedback;
+
+public class FeedbackContentCreator {
+    private StringBuilder stringBuilder;
+    private Feedback feedback;
+
+    public FeedbackContentCreator(Feedback feedback) {
+        this.feedback = feedback;
+        init();
+    }
+
+    /**
+     * Initializes the string buffer object to append content from feedback object
+     */
+    public void init() {
+        stringBuilder = new StringBuilder();
+        stringBuilder.append("== ").append("Feedback for version").append(feedback.getVersion()).append(" ==");
```

Comment:
I believe localization is not needed here, because this ends up on a page where **we** read the feedback.
Devarsh, since this is something that can cause confusion, could you please add a comment explaining this?
`// Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.`